hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -883,6 +885,10 @@ private static StorageContainerLocationProtocol getScmContainerClient(
         false)) {
       rpcServer.refreshServiceAcl(conf, OMPolicyProvider.getInstance());
     }
+
+    rpcServer.addSuppressedLoggingExceptions(OMNotLeaderException.class,
```

Comment:
Good find, so now on server side Rpc does not log these errors.

Reply:
Yes, otherwise we see a lot of noise - "om1 is not the leader" exceptions, on the server side logs if the 1st OM is not the leader.